awswrapper/s3.go

```diff
@@ -313,50 +321,52 @@ func (o *S3Service) UploadToS3Concurrently(content []byte, bucketName string, pa
 			}
 		}
 	}
-	_, err := uploader.Upload(&s3manager.UploadInput{
-		Bucket:      aws.String(bucketName),
-		Key:         aws.String(path),
-		Body:        bodyBytes,
-		ContentType: aws.String(http.DetectContentType(content)),
-		ACL:         aws.String(acl),
-	})
-	fmt.Println("uploading object: ", path)
```

Comment:
Can we preserve this with the log.Printf?

Reply:
This is a deleted line. Logs were converted to `log.Printf` already with the new change

Reply:
Can we restore this log here so that we know `uploader.Upload` was called? In case, `uploader.Upload` panic we won't know it.